src/com/rcv/Logger.java

```diff
@@ -37,6 +37,8 @@
   private static final String TABULATION_LOGGER_NAME = ".tabulation";
   // execution log file name
   private static final String DEFAULT_FILE_NAME = "rcv.log";
+  // first value here is bytes per MB and the second is max MB for the log file
+  private static final Integer TABULATION_LOG_FILE_MAX_SIZE_BYTES = 1000000 * 100;
```

Comment:
nit: if you actually want 100 MB, do 1024 * 1024 * 100

Reply:
https://en.wikipedia.org/wiki/Megabyte

Reply:
That kind of blows my mind too.